docs/source/bibliography.bib

```diff
@@ -352,6 +352,23 @@ @article{RM06
   publisher={IEEE}
 }
 
+@article{SHB21,
+  title = {Randomized algorithms for generalized singular value decomposition with application to sensitivity analysis},
+  author = {Saibaba AK, Hart J, van BloemenWaanders B.},
+  journal = {Numerical Linear Algebra with Applications},
+  volume = {8},
+  issue = {4}, 
+  year = {2021}
+}
+
+@article{SJK15, 
```

Comment:
```suggestion
@article{SJK16, 
```
While the article has been published online 2015, the year of the issue in which it appeared is 2016.